cmd/root.go

```diff
@@ -124,7 +124,7 @@ func init() {
 	rootCmd.Flags().StringP("kubewarden-namespace", "k", defaultKubewardenNamespace, "namespace where the Kubewarden components (e.g. PolicyServer) are installed (required)")
 	rootCmd.Flags().StringP("policy-server-url", "u", "", "URI to the PolicyServers the Audit Scanner will query. Example: https://localhost:3000. Useful for out-of-cluster debugging")
 	rootCmd.Flags().VarP(&level, "loglevel", "l", fmt.Sprintf("level of the logs. Supported values are: %v", logconfig.SupportedValues))
-	rootCmd.Flags().BoolVarP(&printJSON, "print", "p", false, "print result of scan in JSON to stdout")
+	rootCmd.Flags().BoolVarP(&printJSON, "output-scan", "o", false, "output result of scan to stdout in JSON upon completion")
```

Comment:
Nitpicking, can we use `log-fmt`  like [policy server does](https://github.com/kubewarden/policy-server/blob/5f7ddfac5cf3a1af8a83e5f7d559ee3ff67566ec/src/cli.rs#L57-L69)?